x/clp/handler.go

```diff
@@ -0,0 +1,516 @@
+package clp
```

Comment:
Most the core CLP logic/formulas are in this file, and this file together with the keeper is pretty complex. I feel like this could be abstracted away into either the keeper or a separate library containing logic. I'm imagining perhaps that if the keeper had functions for basic symmetric adding/removing of liquidity, create, destroy and basic swap, then the handler would interact with it via these higher level functions rather than through SetPool/SetLP directly.

it also seems like a bad pattern for the handler to have access to SetPool directly.

Reply:
Asana task here: https://app.asana.com/0/1196711589172801/1198989468752757/f